src/ui/components/App.js

```diff
@@ -45,23 +46,43 @@ function setTheme(theme) {
 }
 
 class App extends React.Component {
+  statusRef = React.createRef();
+
   state = {
     orientation: "bottom",
     tooltip: null,
+    recordingId: null,
+    recordingInput: null,
   };
 
   componentDidMount() {
     const { theme } = this.props;
     setTheme(theme);
+
+    const params = new URLSearchParams(document.location.search.substring(1));
+    this.setState({
+      recordingId: params.get("id"),
+    });
```

Comment:
I wonder if we need state here or if we can just do something with an uncontrolled input... not sure

Reply:
yes, this is also part of trying to get that error message within the landing page. i will revert these changes if it is ok to show the error message like it appears on the 2nd attached screenshot.